.gazelcfg.json

```diff
@@ -9,5 +9,8 @@
 		"./federation",
 		"./examples"
 	],
+	"SkippedPaths": [
```

Comment:
for the sources rules to work correctly, we do need to walk the entire repo.

that said, it seems like gazel is fast enough that we could just remove `SrcDirs` and exclude `staging/.*` for kubernetes and get basically the same effect.